website/views/chatbot_api.py

```
@@ -0,0 +1,148 @@
+            reply = ai_answer(message)
+            if reply is not None:
+                return JsonResponse({"reply": reply})
+            # If AI failed → fallback
+            return JsonResponse({"reply": fallback_reply})
+
+        # No API key → fallback
+        return JsonResponse({"reply": fallback_reply})
+
+    except json.JSONDecodeError:
+        return JsonResponse({"reply": "Invalid JSON format."}, status=400)
+    except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)